client/controller/index.node.js

```diff
@@ -30,3 +30,13 @@ const ReduxWrappedLoggedOutLayout = ( { store, primary, secondary } ) => (
  * `context.primary` and `context.secondary` to populate it.
 */
 export const makeLayout = makeLayoutMiddleware( ReduxWrappedLoggedOutLayout );
+
+export function redirectLoggedIn( { isLoggedIn, res }, next ) {
+	// TODO: Make it work also for development env
```

Comment:
Also do you know what is blocking us from enabling `wpcom_user_bootstrap` in all envs? My assumption is that is would solve this issue.